Makefile

```diff
@@ -115,13 +124,37 @@ ifneq ("$(RELEASE)","stable")
 	sed -i '0,/^%define release.*/s/^%define release.*/%define release 0/' inmanta.spec
 endif
 
-	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE),$(ISO_REPO)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR}
-	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE),$(ISO_REPO)" --rebuild ${RPMDIR}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR}
-
-.PHONY: upload
-upload: RPM := $(shell basename ${RPMDIR}/python3-inmanta-${VERSION}-*.x86_64.rpm)
+	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE),$(ISO_REPO)" --buildsrpm --spec inmanta.spec --sources dist --resultdir ${RPMDIR-EL7}
+	mock -r inmanta-and-epel-7-x86_64 --bootstrap-chroot --enablerepo="inmanta-oss-$(RELEASE),$(ISO_REPO)" --rebuild ${RPMDIR-EL7}/python3-inmanta-${VERSION}-*.src.rpm --resultdir ${RPMDIR-EL7}
 
 .PHONY: upload
 upload: ensure-valid-release-type
-	@echo Uploading $(RPM)
-	ssh repomanager@artifacts.ii.inmanta.com "/usr/bin/repomanager --config /etc/repomanager.toml --repo $(REPOMANAGER_REPO) --distro el7 --file - --file-name ${RPM}" < ${RPMDIR}/${RPM}
+# Can be extended later with el8
+	pip3 install cloudsmith-cli
+	@for path_to_rpm in $(shell find rpms-el7 -name '*.x86_64.rpm'); do \
+		rpm=$$(basename $$path_to_rpm) ; \
+		el_version=$$(echo $$rpm| rev| cut -d '.' -f 3| rev |tr -d 'el') ; \
+		if [ "$${RPM_REPOSITORY}" = "iso" ] && [ $${el_version} = "7" ] && [ $${ISO_MAJOR_VERSION} = "3" ]; then \
+        	repomanager@artifacts.ii.inmanta.com "/usr/bin/repomanager --config /etc/repomanager.toml --repo ${REPOMANAGER_REPO} --distro el7 --file - --file-name $${rpm}" < $${path_to_rpm} ; \
```

Comment:
Hard to tell from the GitHub diff but it looks like there are some space-tab inconsistencies here. Iirc it's important for the first indentation to be a tab here, to make sure Make interprets it as part of a recipe (I could be wrong, I always find it confusing). Apart from that, after the initial tab, some lines seem to continue with tabs, others with spaces. Either is fine I think but consistency would be nice, otherwise this might look very unreadable depending on the editor.

Reply:
Nice catch, I haven't noticed it, but it was indeed mixed. I converted them to tabs, thanks.